packages/form-integrations/test/helpers/umbrella-form.js

```diff
@@ -23,9 +25,16 @@ export class UmbrellaForm extends LitElement {
     ));
   }
 
+  /**
+   * @param {string} v
+   */
+  set serializedValue(v) {
+    this.__serializedValue = v;
+  }
+
   render() {
     return html`
-      <lion-form>
+      <lion-form .serializedValue="${this.__serializedValue}">
         <form>
           <lion-fieldset name="full_name">
```

Comment:
I took the same umbrell-form as in the docs, so that it contains all our FormControls